.github/workflows/ask-ai.yml

```diff
@@ -0,0 +1,16 @@
+name: Ask AI
+
+on:
+  pull_request_review_comment:
```

Comment:
This doesn't seem to work on a fork because a PR opened from a fork doesn't have access to secrets 😭😭😭😭😭😭